projects/canopy/src/lib/list/docs/lists.stories.ts

```diff
@@ -234,3 +235,23 @@ lightForegroundListWithIcons.parameters = {
     },
   },
 };
+
+const orderedListTemplate = () => ({
+  template: `
+    <ol lgListWithExpressiveStyling>
+      <li>Item 1</li>
+      <li>Item 2</li>
+      <li>Item 3</li>
+      <li>Item 4</li>
+      <li>Item 5</li>
+      <li>Item 6</li>
+      <li>Item 7</li>
+      <li>Item 8</li>
+      <li>Item 9</li>
+      <li>Item 10</li>
+    </ol>
+  `,
+});
+
+export const orderedListWithNumerals = orderedListTemplate.bind({});
+orderedListWithNumerals.storyName = 'Ordered list with expressive styling';
```

Comment:
Could you please add:

```
orderedListWithNumerals.argTypes = {
  colouredIcons: {
    table: {
      disable: true,
    },
  },
};
```
so that the `colouredIcons` control doesn't appear?
![Screenshot 2024-09-17 at 10 13 19](https://github.com/user-attachments/assets/aa2183a5-9f64-42e2-b4a4-f0d9237d3bea)